github.go

```diff
@@ -11,25 +11,25 @@ import (
 	"golang.org/x/oauth2"
 )
 
-// RateLimitSpecs is an instance of `rateLimitSpecs` that holds RL details received from the API response's headers.
+// rateLimitSpecs is an instance of `rateLimitSpecs` that holds RL details received from the API response's headers.
```

Comment:
Must be capitalized as the exported variable itself. https://blog.golang.org/godoc-documenting-go-code